baselines/Common Controls Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -684,55 +642,19 @@ To configure account conflict management per the policy:
 4.	Select the radio button option: **"Replace conflicting unmanaged accounts with managed ones."**
 5.	Click **Save.**
 
-## 9. Catastrophic Recovery Options for Super Admins
+## 8. Catastrophic Recovery Options for Super Admins
 
 If a catastrophic event occurs in which the GWS Super Admin credentials are lost or stolen, this control is in place to require "break-glass" Super Admin accounts. These accounts are to be physically secured in a highly secure location as a recovery option, with the account self-recovery feature disabled in GWS.
```

Comment:
Group intro needs to be re-written, this is no longer what this group is doing.

Reply:
@adhilto Fixed the intro.